src/main/java/com/devonfw/quarkus/productmanagement/DemoApplication.java

```diff
@@ -1,4 +1,4 @@
-package com.devonfw.demoquarkus;
+package com.devonfw.quarkus.productmanagement;
```

Comment:
This class is also a general class, so do not move it to the productmanagement package